journals.go

```diff
@@ -0,0 +1,79 @@
+package freee
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/google/go-querystring/query"
+	"golang.org/x/oauth2"
+)
+
+const (
+	APIPathJournals = "journals"
+)
+
+type JournalResponse struct {
+	Journal Journal `json:"journals"`
+}
+
+type Journal struct {
+	// 受け付けID
+	ID uint64 `json:"id"`
+	// 事業所ID
+	CompanyID int32 `json:"company_id"`
+	// ダウンロード形式
+	DownloadType string `json:"download_type"`
+	// 取得開始日 (yyyy-mm-dd)
+	StartDate string `json:"start_date"`
+	// 取得終了日 (yyyy-mm-dd)
+	EndDate string `json:"end_date"`
+	// 補助科目やコメントとして出力する項目
+	VisibleTags []string `json:"visible_tags"`
+	// 追加出力するID項目
+	VisibleIDs []string `json:"visible_ids"`
+	// ステータス確認用URL
+	StatusURL string `json:"status_url"`
+	// 集計結果が最新かどうか
+	UpToDate bool `json:"up_to_date"`
+	// 集計が最新でない場合の要因情報
+	UpToDateReasons []JournalUpToDateReasons `json:"up_to_date_reasons"`
+}
+
+type JournalUpToDateReasons struct {
+	// コード
+	Code string `json:"code"`
+	// 集計が最新でない理由
+	Message string `json:"message"`
+}
+
+type GetJournalsOpts struct {
+	// ダウンロード形式
+	DownloadType string `json:"download_type"`
```

Comment:
@shirakiyo `GetXXXOpts` のtagは `url` になるかと思います:bow:

```suggestion
	DownloadType string `url:"download_type"`
```

Reply:
ありがとうございます:pray: